packages/cozy-konnector-libs/libs/linkBankOperations.js

```diff
@@ -64,21 +66,63 @@ class Linker {
       operationId: matchingOperation && matchingOperation._id
     })
 
-    return this.cozyClient.data.updateAttributes(
+    return this.updateAttributes(
       DOCTYPE_OPERATIONS,
-      debitOperation._id,
+      debitOperation,
       { reimbursements: reimbursements }
     )
   }
 
+  /* Buffer update operations */
+  updateAttributes (doc, attrs) {
+    Object.assign(doc, attrs)
+    this.toUpdate.push(doc)
+    return Promise.resolve()
+  }
+
+  commitChanges () {
+    return cozyClient.fetchJSON(
+      'POST',
+      `data/${DOCTYPE_OPERATIONS}/_bulk_docs`,
+      {
+        data: JSON.stringify
+      }
+    )
+  }
+
+  getOptions (opts={}) {
+    const options = {...opts}
+
+    if (typeof options.identifiers === 'string') {
+      options.identifiers = [options.identifiers.toLowerCase()]
+    } else if (Array.isArray(options.identifiers)) {
+      options.identifiers = options.identifiers.map(id => id.toLowerCase())
+    } else {
+      throw new Error(
+        'linkBankOperations cannot be called without "identifiers" option'
+      )
+    }
+
+    options.amountDelta = options.amountDelta || DEFAULT_AMOUNT_DELTA
+    options.minAmountDelta = options.minAmountDelta || options.amountDelta
+    options.maxAmountDelta = options.maxAmountDelta || options.amountDelta
```

Comment:
Can `options.amountDelta`, `options.minAmountDelta` or `options.maxAmountDelta` be `0` ? If yes, you may end with `options.minAmountDelta === options.amountDelta` where you wanted it to be `0` initially.

Reply:
Thanks for the catch

Reply:
I used defaults from lodash since `options.minAmountDelta = options.minAmountDelta !== undefined ? options.minAmountDelta : DEFAULT_MIN_AMOUNT_DELTA` is too verbose for my taste.